packages/lib/src/interactions/box.test.ts

```diff
@@ -0,0 +1,203 @@
+import { Vector3 } from 'three';
+
+import Box from './box';
+
+const ZERO_COORDS = { x: 0, y: 0, z: 0 };
+
+describe('size', () => {
+  it('should compute box size', () => {
+    const box1 = Box.empty();
+    expect(box1.size).toEqual({ width: 0, height: 0 });
+
+    const box2 = Box.empty(new Vector3(10, 10));
+    expect(box2.size).toEqual({ width: 0, height: 0 });
+
+    const box3 = Box.empty().expandByPoint(new Vector3(10, -5));
```

Comment:
Could we use `expandBySize` here instead of `expandByPoint` to make the test easier to read ?

Reply:
Sure but since you are only checking `size` here, I would find it easier to have something like:

```ts
const newSize = { width: 10, height: 20 }
const box3 = Box.empty().expandBySize(newSize);
expect(box3.size).toEqual(newSize);
```